api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/handler/AvailabilityHandler.java

```diff
@@ -466,4 +471,126 @@ public void getMetricStats(
                 .map(ApiUtils::collectionToResponse)
                 .subscribe(asyncResponse::resume, t -> asyncResponse.resume(serverError(t)));
     }
+
+    @GET
+    @Path("/aggregated")
```

Comment:
I'm not sure if "stats" fits well. For now "stats" is about time-aggregation, with buckets and so on, whereas here it's about multiple series aggregation, without any time bucket. I fear it would be confusing if we use the same word.

On hawkular-dev ML Jay proposed: "/aggregate, /set, /group or whatever" ... is there anyone of these you find better? Or maybe something like "/merged" ?